src/egherkin.app.src

```diff
@@ -0,0 +1,12 @@
+{application, egherkin,
```

Comment:
Is it possible to have ebin/egherkin.app generated by rebar3 without this? With erlang.mk, this file is optional and erlang.mk automates the generation of the .app file from all the info available in the makefile

Reply:
I'll have a look.

Reply:
I checked rebar3's documentation. My understanding is that the .app.src file is needed by rebar to generate the .app file.